backend/indexer/ponder.config.ts

```diff
@@ -0,0 +1,137 @@
+    polygon: { chainId: 137, transport: http((SupportedNetworks.find((chain) => chain.chainId == 137))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    opGoerli: { chainId: 420, transport: http((SupportedNetworks.find((chain) => chain.chainId == 420))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    klaytnTestnet: { chainId: 1001, transport: http((SupportedNetworks.find((chain) => chain.chainId == 1001))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    mantle: { chainId: 5000, transport: http((SupportedNetworks.find((chain) => chain.chainId == 5000))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    mantleTestnet: { chainId: 5001, transport: http((SupportedNetworks.find((chain) => chain.chainId == 5001))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    klaytn: { chainId: 8217, transport: http((SupportedNetworks.find((chain) => chain.chainId == 8217))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    base: { chainId: 8453, transport: http((SupportedNetworks.find((chain) => chain.chainId == 8453))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    arbitrum: { chainId: 42161, transport: http((SupportedNetworks.find((chain) => chain.chainId == 42161))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    avalanche:  { chainId: 43114, transport: http((SupportedNetworks.find((chain) => chain.chainId == 43114))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    linea: { chainId: 59144, transport: http((SupportedNetworks.find((chain) => chain.chainId == 59144))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    mumbai: { chainId: 80001, transport: http((SupportedNetworks.find((chain) => chain.chainId == 80001))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 1 },
+    baseGoerli: { chainId: 84531, transport: http((SupportedNetworks.find((chain) => chain.chainId == 84531))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    arbitrumGoerli: { chainId: 421613, transport: http((SupportedNetworks.find((chain) => chain.chainId == 421613))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    scrollSepolia: { chainId: 534351, transport: http((SupportedNetworks.find((chain) => chain.chainId == 534351))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+    sepolia: { chainId: 11155111, transport: http((SupportedNetworks.find((chain) => chain.chainId == 11155111))?.bundler), maxHistoricalTaskConcurrency: 1, maxRequestsPerSecond: 10 },
+  },
+  contracts: {
+    EtherspotPaymaster: {
+      abi: EtherspotPaymasterAbi,
+      network: {
+        mainnet: {
+          address: '0x7F690e93CecFca5A31E6e1dF50A33F6d3059048c',
```

Comment:
Can the address and blocknumber be configurable variables? These values can be the defaults.

Reply:
the address couldn't be given from the config.json since it reads as string | undefined which doesn't allow ponder as the formatted string should be of readonly `0x` prefixed. Thats why gave the address as a string